### PR TITLE
fix(agnostic): prefer manager built-in version checker and suggest `arm-v8a` arch

### LINKED ISSUE #12: Suggestions / Issues for the general step-by-step guide
1. The [Checking versions](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/step-by-step-general/02-check-ver/) step is unnecessary. In step 3, the guide should recommend using the feature in the manager that opens a search in the browser for the suggested version. So the user doesn't need to know the suggested version.

2. In step 3 (Getting the files ready), there is no information on downloading the apk variant supported by the arch of the device. For example, you don't want users downloading the x86 variant of a YTM apk.

### PR DESCRIPTION
**Pull request by SodaWithoutSparkles**
fixes #12 @KobeW50 

* requests landing on "Checking versions" page would now be redirected.
* since `arm-v7` users can't use manager anyway, suggest `arm-v8a` arch apk
* clarify on the meaning of "Suggested: Any version"
* shifts number in title by -1

> [!IMPORTANT]  
> The number shift in title causes a de-sync between filename and title. This is done to ensure old links still works.

### CODE REVIEW

File: step-by-step-general/03-get-files.md [hunk 1]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
+!!! If it said "Any version"
+This means that nobody has reported a version that did not work *yet*.
+It means that you can *try* the latest version of apk available.
+In case it doesn't work, go back a few versions and check if it works, and repeat if it doesn't.
+You can also report back the version that works for you at [Discord](https://discord.gg/revanced) or [Github (if nobody else has reported)](https://github.com/ReVanced/revanced-patches/issues)
 !!!
-3. Get the APK, APKMirror usually has them.
-4. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
+5. You can tap the version shown to open an online search result of the suggested apk. You can also just google it.
+6. If there are multiple variants, you most likely need the one marked `arm64-v8a` and `APK` (not bundle).
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:33]: Combining the arch information with the file type can cause confusion. Separating them may be simpler:

```suggestion
6. If there are multiple variants, choose the one marked `APK` (not bundle) with the "arch" marked as universal. If no universal arch is available, you most likely need the `arm64-v8a` or `armeabi-v7a` one.
```

File: step-by-step-general/03-get-files.md [hunk 2]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
+4. The suggested version of the app would be shown. 
+!!! If it said "Any version"
+This means that nobody has reported a version that did not work *yet*.
+It means that you can *try* the latest version of apk available.
+In case it doesn't work, go back a few versions and check if it works, and repeat if it doesn't.
+You can also report back the version that works for you at [Discord](https://discord.gg/revanced) or [Github (if nobody else has reported)](https://github.com/ReVanced/revanced-patches/issues)
 !!!
-3. Get the APK, APKMirror usually has them.
-4. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
+5. You can tap the version shown to open an online search result of the suggested apk. You can also just google it.
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:32]: Maybe recommend the search result from apkmirror.com

File: step-by-step-general/03-get-files.md [hunk 3]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
+6. If there are multiple variants, you most likely need the one marked `arm64-v8a` and `APK` (not bundle).
+=== Image for reference
+![This variant is both `arm-v8a` and is an apk](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/022-verify-apk-correct-arch?raw=true)
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:35]: The screenshot doesn't highlight that this one is an apk and not a bundle

Comment (SodaWithoutSparkles) [reply step-by-step-general/03-get-files.md]: I thought I have drawn a blue box over it... 
maybe I will draw one more arrow

File: step-by-step-general/03-get-files.md (hunk 2 above)
Comment (SodaWithoutSparkles) [reply step-by-step-general/03-get-files.md:32]: good idea

File: step-by-step-general/03-get-files.md [hunk 4]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
 
-2. Google search the app you wanted to patch and the version required. Example: "youtube music 6.10.52 apk".
-!!!warning
-**Do NOT install the downloaded APK.**
+If you are certain that you have the app installed, using the search bar may help.
 !!!
-!!!warning
-**The app from Google Play may NOT work.** You may or may not be able to patch the one from Google Play. Use the downloaded one.
+5. The suggested version of the app would be shown. 
+!!! If it said "Any version"
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:26]:
```suggestion
5. The suggested version of the app will be shown. 
!!! If it says "Any version"
```

File: step-by-step-general/03-get-files.md [hunk 5]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
+This means that nobody has reported a version that did not work *yet*.
+It means that you can *try* the latest version of apk available.
+In case it doesn't work, go back a few versions and check if it works, and repeat if it doesn't.
+You can also report back the version that works for you at [Discord](https://discord.gg/revanced) or [Github (if nobody else has reported)](https://github.com/ReVanced/revanced-patches/issues)
 !!!
-3. Get the APK, APKMirror usually has them.
-4. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
+6. You can tap the version shown to open an online search result of the suggested apk. APKMirror usually has them. You can also just google it.
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:32]:
```suggestion
6. You can tap the version shown to open an online search result of the suggested apk. Or you can google it. APKMirror will usually have the apk.
```

File: step-by-step-general/03-get-files.md [hunk 6]
```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
...
+7. If there are multiple variants, choose the one marked `APK` (not bundle) with the "arch" marked as universal. If no universal arch is available, you most likely need the `arm64-v8a` or `armeabi-v7a` one.
+=== Image for reference
+![This variant is both `arm-v8a` and is an apk](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/022-verify-apk-correct-arch.jpg?raw=true)
+===
+8. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
```
Comment (KobeW50) [inline step-by-step-general/03-get-files.md:37]:
```suggestion
8. Verify that it says "DOWNLOAD APK" and not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
```

**Pull request merged**